### ISSUE TITLE
Pattern Capacity Card EU Usage Stacking Incorrectly

### ISSUE THREAD

**Issue by Cleista** [GTNewHorizons/GT-New-Horizons-Modpack#6345]
#### Which modpack version are you using?
2.0.9.0QF4

#
#### If in multiplayer; On which server does this happen? 
Private server

#
#### What did you try to do, and what did you expect to happen?
Queue crafting command for Quantumprocessor Mainframe via AE2 with sided ME interface pointing at input bus of circuit assembler processing array (all sub-component circuit have recipes that share the same interface).

ME interface uses 5EU/t, since it has 3 Pattern Capacity Cards (x64 power usage) I expect an energy drain increase of **320 EU/t**. 

#
#### What happened instead? (Attach screenshots if needed)
[Energy drain increases to at least 100k EU/t](https://cdn.discordapp.com/attachments/181078474394566657/727270576351215646/unknown.png). It's probably even higher, but I guess ME system won't show a number higher than my AE energy storage capacity / 20 ticks per second. 

#
#### What do you suggest instead/what changes do you propose?
Fix energy consumption calculation so it does not scale incorrectly and kill people's ME systems.

**Comment by Cleista** [comment]
I was able to reproduce this again and found that this seems to be related to **blocking mode** being used on the ME interface. 

Looking at [the source](https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/blob/rv3-1.7.10/src/main/java/appeng/me/cluster/implementations/CraftingCPUCluster.java#L641), the power drain is `anInput.getStackSize() * Math.pow(4.0, ((DualityInterface)m).getInstalledUpgrades(Upgrades.PATTERN_CAPACITY));` which gives me 48640 EU/t drain trying to craft a Quantumprocessor Mainframe. 

When I queue only one of the item, ingredients go into the input bus immediately and the power spike is instantaneous and doesn't affect my system. However, when I queue multiple and the interface is blocked on crafting the subcomponents (e.g. the 3rd and 4th Master Quantumcomputers while the ingredients for the first Mainframe is ready), it drains this amount of energy _every tick_ (and it's like that for all the subcomponents) until the input bus is empty and the ingredients can go in. 

This is quite counterintuitive and was not at all obvious until I did a lot of debugging, so it should be changed to a much more reasonable drain per tick while waiting on a craft instead of draining a massive amount of energy only when trying to insert ingredients into the machine.
